src/test/javascript/spec/component/overview/course-faq/course-faq.component.spec.ts

```diff
@@ -156,4 +159,26 @@ describe('CourseFaqs', () => {
         courseFaqComponentFixture.detectChanges();
         expect(alertServiceStub).toHaveBeenCalledOnce();
     });
+
+    it('should call sortService when sortRows is called', () => {
+        jest.spyOn(sortService, 'sortByProperty').mockReturnValue([]);
+        courseFaqComponent.sortFaqs();
+        expect(sortService.sortByProperty).toHaveBeenCalledOnce();
+    });
+
+    it('should scroll and focus on the faq element with given id', () => {
+        const nativeElement = {
+            id: 'faq-1',
+            scrollIntoView: jest.fn(),
+            focus: jest.fn(),
+        };
+        const elementRef = new ElementRef(nativeElement);
+        const queryList = new QueryList<ElementRef>();
+        queryList.reset([elementRef]);
+        courseFaqComponent.faqElements = queryList;
+        courseFaqComponent.scrollToFaq(1);
+
+        expect(nativeElement.scrollIntoView).toHaveBeenCalledWith({ behavior: 'smooth', block: 'start' });
+        expect(nativeElement.focus).toHaveBeenCalled();
```

Comment:
Could we be more specific here?

```suggestion
        expect(nativeElement.scrollIntoView).toHaveBeenCalledExactlyOnceWith({ behavior: 'smooth', block: 'start' });
        expect(nativeElement.focus).toHaveBeenCalledOnce();
```